src/uwtools/drivers/mpas_base.py

```diff
@@ -114,12 +115,22 @@ def streams_file(self):
         yield self._taskname(fn)
         path = self._rundir / fn
         yield asset(path, path.is_file)
-        yield file(path=Path(self._driver_config["streams"]["path"]))
-        render(
-            input_file=Path(self._driver_config["streams"]["path"]),
-            output_file=path,
-            values_src=self._driver_config["streams"]["values"],
-        )
+        yield None
+        streams = Element("streams")
+        for item in self._driver_config["streams"]:
+            stream = SubElement(streams, "stream" if item["mutable"] else "immutable_stream")
+            for attr in ["name", "type", "filename_template"]:
```

Comment:
Although attribute order is not semantically meaningful in XML, `lxml` appears to output attributes in the order they are defined, so I've tried to order the generation of attributes in the `for` loops here to match the MPAS docs and examples.